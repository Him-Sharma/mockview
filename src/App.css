.App {
  text-align: center;
}
.header {
  width: 100%;
  color: white;
  font-size: calc(10px + 2vmin);
  background-color: #41495a;
}
.control {
  width: 100%;
  margin-top: 40px;
  margin-bottom: 40px;
  background-color: #41495a;
}
.control > input{
  margin: 10px;
}
.flex-container {
  display: flex;
  flex-direction: row;
  justify-content: flex-start;
  align-items: center;
  background-color: #41495a;
}
.flex-container > div {
  color: white;
  margin: 10px;
  width: 33%;
  height: 100%;
}
