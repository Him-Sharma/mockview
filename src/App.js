import React from "react";
import "./App.css";
import MockView from "./containers/MockView/MockView";

function App() {
  return (
    <div className="App">
      <MockView />
    </div>
  );
}

export default App;
