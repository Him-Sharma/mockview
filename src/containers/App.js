import React from "react";
import "./App.css";
import Header from "../components/Header/Header";
import Control from "../components/Control/Control";
import StubList from "../components/StubList/StubList";

function App() {
  return (
    <div className="App">
      <Header />
      <Control />
      <StubList />
    </div>
  );
}

export default App;
