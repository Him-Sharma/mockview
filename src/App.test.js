import React from "react";
import { render } from "@testing-library/react";
import App from "./App";

test("renders mockview header", () => {
  const { getByText } = render(<App />);
  const headerElement = getByText("Mockview");
  expect(headerElement).toBeInTheDocument();
});
