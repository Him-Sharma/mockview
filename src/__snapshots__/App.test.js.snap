// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<body>
  <div>
    <div
      class="App"
    >
      <div
        class="MockView"
      >
        <h1>
          Mockview
        </h1>
        <section
          class="Control"
        >
          <input
            data-testid="serverUrlInput"
            id="serverUrlInput"
            type="text"
          />
          <button
            data-testid="fetchStubButton"
          >
            Fetch Stubs
          </button>
        </section>
        <section
          class="StubContainer"
        >
          <h1>
            No Stub items to display
          </h1>
        </section>
      </div>
    </div>
  </div>
</body>
`;
